### PR TITLE
add new 2 lines

### PR DESCRIPTION
**Pull request by Skelman50**
add new 2 lines

### CODE REVIEW

File: newfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 adadadad
 asd
-asd
\ No newline at end of file
+asd
```
Comment (Boriska-byte) [inline newfile:3]: what is this?

Comment (Skelman50) [reply newfile:3]: this is a new asd

Comment (Boriska-byte) [reply newfile:3]: add asdf

File: newfile [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+123
+123
+asdf
```
Comment (Boriska-byte) [inline newfile:6]: asdfg

**Pull request merged**